app/controllers/issues_controller.rb

```diff
@@ -31,6 +31,24 @@ def create
     end
   end
 
+  def getIssues
+    category = params[:category]
+    subCategory = params[:sub_category]
```

Comment:
we don't follow camel case in rails. please follow snake case =>i.e neeraj_kumar